script/git-ship

```diff
@@ -1,26 +1,42 @@
 #!/usr/bin/env perl
 use Applify;
 use App::git::ship;
+use Mojo::Util 'monkey_patch';
 
 documentation 'App::git::ship';
 version 'App::git::ship';
 
-app {
-  my $self   = shift;
-  my $action = shift || 'ship';
-  my $ship   = App::git::ship->new;
+subcommand build    => 'Build the project'          => _applify('build');
+subcommand clean    => 'Clean unwanted files'       => _applify('clean');
+subcommand coverage => 'Test coverage'              => _applify('test_coverage');
+subcommand start    => 'Start a project'            => sub { };
+subcommand upload   => 'Upload project to git/cpan' => _applify('ship');
+
+sub _applify {
```

Comment:
This function name is not very descriptive. Please review https://github.com/jhthorsen/app-git-ship/pull/26/files#r309385852 again.